cfme/fixtures/v2v.py

```diff
@@ -41,6 +60,183 @@ def conversion_tags(appliance, host_creds):
     tag2 = appliance.collections.categories.instantiate(
         display_name='V2V - Transformation Method').collections.tags.instantiate(
         display_name='VDDK')
-    host_creds.add_tags(tags=(tag1, tag2))
+    if _tag_cleanup(host_creds, tag1, tag2):
+        host_creds.add_tags(tags=(tag1, tag2))
     yield
     host_creds.remove_tags(tags=(tag1, tag2))
+
+
+def get_vm(request, appliance, second_provider, template, datastore='nfs'):
+    source_datastores_list = second_provider.data.get('datastores')
```

Comment:
`.get('datastores', [])` - The default for a .get that doesn't find the key is `None`, which will cause an exception on the next line because you can't iterate over `NoneType`